Readme.md

```diff
@@ -18,6 +18,7 @@ This is a Gatsby generated site pulling data from:
 - Package dependencies (Windows)
   - Python 3
   - Node-gyp - see https://github.com/nodejs/node-gyp#on-windows
+  - Install Visual Studio 2022 - Desktop Development with C++ in the Visual Studio Installer
```

Comment:
Good call, I know about this one and its pretty obscure.

Maybe add a screenshot as well